### DISCUSSION TITLE
Connecting User Location to AD

### DISCUSSION THREAD

**Discussion by Oddalmighty** [snipe/snipe-it#10270] [Q&A]
We use the Office field (physicalDeliveryOfficeName) in Active Directory to separate our staff into locations. The only staff that are in specific containers or OUs are our WAN staff, so current LDAP location methods will not work for all of our staff. We have a location created in Snipe-IT for each of our offices (there are over 100). I'm trying to find a way to connect the physicalDeliveryOfficeName in AD to Snipe-IT users via LDAP. Is there a way to do this? That way when we update someone's location in AD, it will populate in Snipe-IT automatically. Ultimately, I would like to see assets move with staff when they change offices, but I know there are other requests open for that feature / issue already.

Thank you in advance!

**Comment by TechBreakHelp** [comment]
Snipe-It has it’s own LDAP connector which supports AD. Have a look to https://snipe-it.readme.io/docs/ldap-sync-login

**Reply by Oddalmighty** [reply]
I have LDAP working already (I should have included that). I have all users syncing correctly from AD to Snipe-IT. The only thing I'm missing is the location which I would like to pull from physicalDeliveryOfficeName so that AD locations would match up with the Snipe-IT locations we've created. Using the LDAP connector, how would I put that in?

**Reply by TechBreakHelp** [reply]
Have a look to this discussion https://github.com/snipe/snipe-it/issues/3993

**Reply by Oddalmighty** [reply]
That one is using separate OUs. Our staff are separated into OUs by their department, not location. Their location is in the office field. It looks like that discussion was closed as well, though some had similar questions.
